core/systemcontracts/upgrade.go

```diff
@@ -634,8 +636,8 @@ func init() {
+				CommitUrl:    "https://github.com/bnb-chain/bsc-genesis-contract/commit/a803125f635f315e332fd5164e91a3df1d48559c",
```

Comment:
Please change the commit id of other contract code as well.

Reply:
Done